fix(CLI): Clarify note re: updating/uninstalling package-manager-installed instances of `sentry-cli` (#4219)
getsentry/sentry-docs: src/docs/product/cli/installation.mdx

## Updating and Uninstalling

> You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the sentry command line interface. These commands might be unavailable in certain situations, generally when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.

Reviewer: ```suggestion
You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the Sentry command line interface (CLI). These commands might be unavailable in certain situations, such as when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working, and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.
```

Author: Hmmm... I said "generally" in the sense of "usually," "most of the time," etc. I feel like "such as" is more "sometimes" than "most of the time." I did fix up the end of the first sentence, though.

```suggestion
You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the `sentry-cli` binary. These commands may be unavailable in certain situations, generally when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.
```
